archetypes/archetype-app-quickstart/src/test/resources/projects/ssoidc/reference/README.md

```diff
@@ -0,0 +1,53 @@
+# App
+
+This project contains a maven application with [AWS Java SDK 2.x](https://github.com/aws/aws-sdk-java-v2) dependencies.
```

Comment:
Do we say that it's using SSO/SSOOIDC or are the other projects also generic?

Reply:
We don't specify which individual dependencies that are included for other property of the archetype, so I think we should leave it as-is